Source/WebCore/platform/mediastream/gstreamer/GStreamerAudioCaptureSource.h

```diff
@@ -46,6 +46,9 @@ class GStreamerAudioCaptureSource : public RealtimeMediaSource, public ThreadSaf
     ThreadSafeWeakPtrControlBlock& controlBlock() const final { return ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<GStreamerAudioCaptureSource>::controlBlock(); }
     virtual ~GStreamerAudioCaptureSource();
 
+    // GStreamerCapturer::Observer
```

Comment:
.

Reply:
This is not a sentence. :)

Reply:
imho this guideline applies here: https://webkit.org/code-style-guidelines/#comments-messages-disambiguation
it's similar to closing namespace comments which don't require a full stop either.